Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -44,7 +44,7 @@ def test_validate_all_fields_of_parsed_metadata(self, dummy_pack_metadata):
         assert parsed_metadata['price'] == 0
         assert parsed_metadata['serverMinVersion'] == '5.5.0'
         assert parsed_metadata['currentVersion'] == '2.3.0'
-        assert parsed_metadata['tags'] == ["Tag Number One", "Tag Number Two"]
+        assert parsed_metadata['tags'] == ["tag number one", "Tag number two"]
```

Comment:
why one is with cap and the other don't?

Reply:
it is unit test, input has same values